apps/designer/app/shared/router-utils/index.ts

```diff
@@ -1 +1 @@
-export * from "./canvas-request-params";
+export * from "./user-content-params";
```

Comment:
user content is what we call "build", right?

Reply:
Not sure. Do you mean something related to Build type in database?

In this PR it means content at canvas / preview. We can use any term, but "build" is not intuitive for me at all, at least right now.

Reply:
no. the entire thing on canvas is user's "build", tat's how we call it so far in a lot of places, it's what user builds

Reply:
Ok. Still not intuitive for me to be honest 😅  
But now I know, and will rename here.